src/components/parsero/parsero.tsx

```diff
@@ -1,111 +1,177 @@
-import { Button, LoadingOverlay, Stack, TextInput } from "@mantine/core";
+import { Button, Stack, TextInput } from "@mantine/core";
 import { useForm } from "@mantine/form";
-import { useCallback, useState } from "react";
+import { useCallback, useState, useEffect } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
 import { UserGuide } from "../UserGuide/UserGuide";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
+import InstallationModal from "../InstallationModal/InstallationModal";
+import { RenderComponent } from "../UserGuide/UserGuide";
 
-const title = "Parsero";
-const description =
-    "Parsero is a free script written in Python which reads the Robots.txt file of a web server and looks at the Disallow entries." +
-    "The Disallow entries tell the search engines what directories or files hosted on a web server mustn’t be indexed. " +
-    "\n\nParsero Reference Guide: https://www.kali.org/tools/parsero/ \n\n" +
-    "How to use Parsero:\n\n" +
-    "Step 1: Enter an URL.\n" +
-    "       E.g. www.google.com\n\n" +
-    "Step 2: Click Start Parsero" +
-    "\n\n Step 3 : View Results";
+/**
+ * Represents the form values for the DNSMap component.
+ */
 interface FormValues {
```

Comment:
The name of the interface should be FormValuesType. 
```suggestion
interface FormValuesType {
```